jobs/pgbackup/spec

```diff
@@ -27,22 +27,21 @@ properties:
     default: pgdumpall
   pgbackup.host:
     description: "Hostname or IP address to connect to PostgreSQL on"
-    default: "localhost"
+    default: "127.0.0.1"
   pgbackup.port:
     description: "TCP port to connect to PostgreSQL on"
     default: 5432
   pgbackup.username:
     description: "User to connect to PostgreSQL as"
-    default: "pgbackup"
+    default: "PLEASE-CHANGE-THIS"
   pgbackup.password:
     description: "Password to authenticate to PostgreSQL with"
     default: "PLEASE-CHANGE-THIS"
   pgbackup.version:
     description: "Version of PostgreSQL to be backed up/restored"
     default: "9.5"
-
-  pgbackup.pgdump.arguments:
-    description: "Extra arguments to pass to PostgreSQL's pg_dump utility"
+  pgbackup.arguments:
+    description: "Extra arguments to pass to PostgreSQL's pg_dumpall utility"
```

Comment:
"Extra arguments to pass to PostgreSQL's the dump command utility specified in pgbackup.cmdname property"